src/StructuredDiscussions/TopicRepository.php

```diff
@@ -49,4 +49,28 @@ public function getByTitle( Title $title ): ?SDTopic {
 
 		return new SDTopic( $viewTopic['topic'] );
 	}
+
+	public function getByOwner( Title $title ): array {
+		$parameters = [ 'page' => $title->getTalkPage()->getFullText() ];
```

Comment:
`getTalkPage()` is deprecated.

```suggestion
                $talkPage = $title->getTalkPageIfDefined();
                
                if ( !$talkPage ) {
                    return [];
                }
                
		$parameters = [ 'page' => $talkPage->getFullText() ];
```